osquery/tables/applications/browser_utils.cpp

```diff
@@ -168,7 +210,22 @@ void genExtension(const std::string& uid,
 
   r["identifier"] = fs::path(path).parent_path().parent_path().leaf().string();
   r["path"] = path;
-  results.push_back(r);
+  r["profile"] = profile_name;
+
+  const std::vector<std::vector<std::string>> script_list =
+      genContentScriptDetail(kScriptKey, tree);
+  const std::vector<std::vector<std::string>> matches_list =
+      genContentScriptDetail(kMatchesKey, tree);
+
+  for (int i = 0; i < script_list.size(); i++) {
```

Comment:
I'd like to test this implementation locally but I think we should explore an alternative. Instead of having the existing table with duplicated rows we should introduce a new table that reports these matches. Then a user can join them together.

Reply:
Yeah, I agree. I'll roll this implementation into a new table. It looks like the Chrome extension ID is a decent candidate for joins.

Reply:
Actually, it probably makes sense to have two separate tables here -- `chrome_extension_scripts` and `chrome_extension_matches`, since the two are functionally disjoint.